samcli/local/docker/lambda_build_container.py

```diff
@@ -142,12 +162,14 @@ def _make_request(
                         "dependency_manager": dependency_manager,
                         "application_framework": application_framework,
                     },
+                    "base_dir": container_dirs["base_dir"],
```

Comment:
This json is used to call AWS Lambda Builders in an RPC mode within the container. `base_dir`, `specified_workflow`, and `mount_with_write` are not apart of that: https://github.com/aws/aws-lambda-builders/blob/develop/aws_lambda_builders/__main__.py#L117

Was this meant to be added here?

Reply:
Remove if that's the only consumer.